blockscout-ens/graph-node/subgraphs/zns-subgraph/src/zns-registry.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  MintedDomain as MintedDomainEvent,
+  PrimaryDomainSet as PrimaryDomainSetEvent,
+  RenewedDomain as RenewedDomainEvent,
+  Transfer as TransferEvent,
+} from "../generated/ZNSRegistry/ZNSRegistry";
+import {
+  Domain,
+  Account,
+  Transfer,
+  Registration,
+  ExpiryExtended,
+} from "../generated/schema";
+import { crypto, Bytes } from "@graphprotocol/graph-ts";
+
+export function handleMintedDomain(event: MintedDomainEvent): void {
+  let domain = new Domain(event.params.tokenId.toString());
+  domain.name = event.params.domainName;
+  domain.createdAt = event.block.timestamp;
+  domain.expiryDate = event.params.expiry;
+  domain.labelName = event.params.domainName;
```

Comment:
`label` should be different from `name` 💯 

`name` -- `vitalik.eth`
`label` -- `vitalik`

https://docs.ens.domains/terminology#label

Reply:
Ok we'll make sure that is done